org.mixedrealitytoolkit.uxcore/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "org.mixedrealitytoolkit.uxcore",
-  "version": "3.1.0-development",
+  "version": "3.1.1-development",
```

Comment:
I think the package dependencies need to be updated throughout as well, since this depends on a new API in the Core package.

Reply:
good point!

Reply:
Good call!